libraries/AP_HAL_ChibiOS/Util.cpp

```diff
@@ -27,6 +27,7 @@
 #include "hwdef/common/flash.h"
 #include <AP_ROMFS/AP_ROMFS.h>
 #include <AP_Common/ExpandingString.h>
+#include <AP_InternalError/AP_InternalError.h>
```

Comment:
The InternalError is only used when the condition `#if defined(HAL_DEBUG_BUILD) && !defined(IOMCU_FW)` is met. Is it preferred this is guarded by the same? Or happy to just include as is?

https://github.com/ArduPilot/ardupilot/pull/25033/files#diff-a4a3d06086bfc0e82120db26f5962d8ffc4f623e8e3090a2604e29b78c2225b1R141